packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/utils/normalizeRelations.js

```diff
@@ -26,12 +26,21 @@ export const normalizeRelations = (
   relations,
   { modifiedData = {}, shouldAddLink = false, mainFieldName, targetModel }
```

Comment:
yep good idea!

Reply:
🙇🏻‍♀️ danke

Reply:
I added some doc [here](https://github.com/strapi/strapi/pull/14527/commits/fc6e4adbee2eeed292bd0c39816adea9cdf6bf70)
not used to document them this way, let me know if I can improve it
I left a `TO FIX` on it because I didn't fully understand `shouldAddLink` logic